.github/workflows/review-delete-workflow.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - id: get-branch-name-sanitized
         name: Sanitize branch name
         shell: bash
-        run: echo "::set-output name=branch::$(jq --raw-output .pull_request.head.ref "$GITHUB_EVENT_PATH" | tr -cd '[a-zA-Z0-9]_-')"
+        run: echo "::set-output name=branch::$(jq --raw-output .pull_request.head.ref "$GITHUB_EVENT_PATH" | tr -cd '[a-zA-Z0-9]-')"
```

Comment:
Won't this still break if it has an underscore? is the `[a-zA-Z0-9]-` the regex for the valid branch name? if so this will still break no?